config/printer-biqu-b1-se-plus.cfg

```diff
@@ -0,0 +1,219 @@
+########################################
+[stepper_x]
+step_pin: PE2
+dir_pin: !PE1
+enable_pin: !PE3
+microsteps: 16
+rotation_distance: 40.12
+endstop_pin: !PC1
+position_endstop: 0
+position_max: 310
+homing_speed: 50
+#homing_positive_dir: false
+#homing_retract_dist: 5
+
+[tmc2208 stepper_x]
+uart_pin: PE0
+run_current: 0.800
+hold_current: 0.400
+stealthchop_threshold: 999999
+
+########################################
+# Stepper Y Pins and TMC2208 configuration
+########################################
+[stepper_y]
+step_pin: PD5
+dir_pin: PD4
+enable_pin: !PD6
+microsteps: 16
+rotation_distance: 40.28
+endstop_pin: !PC3
+position_endstop: 0
+position_max: 310
+homing_speed: 50
+#homing_retract_dist: 5
+
+[tmc2208 stepper_y]
+uart_pin: PD3
+run_current: 0.800
+hold_current: 0.400
+stealthchop_threshold: 999999
+
+########################################
+# Stepper Z Pins and TMC2208 configuration
+########################################
+[stepper_z]
+step_pin: PA15
+dir_pin: PA8
+enable_pin: !PD1
+microsteps: 16
+rotation_distance: 8.13656
+endstop_pin: probe:z_virtual_endstop
+homing_speed: 10
+second_homing_speed: 1
+position_min: -2
+position_max: 340
+
+[tmc2208 stepper_z]
+uart_pin: PD0
+run_current: 0.800
+hold_current: 0.400
+stealthchop_threshold: 999999
+
+########################################
+# Extruder Pins and TMC2208 configuration
+########################################
+[extruder]
+step_pin: PD15
+dir_pin: !PD14
+enable_pin: !PC7
+microsteps: 16
+rotation_distance: 34.2152 # Calibrar - ver https://www.klipper3d.org/Rotation_Distance.html
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PB3
+sensor_type: Generic 3950
+sensor_pin: PA2 #thermistor pin
+control: pid
+pid_Kp: 22.2
+pid_Ki: 1.08
+pid_Kd: 114
+min_temp: 0
+max_temp: 250
+
+[tmc2208 extruder]
+uart_pin: PC6
+run_current: 0.800
+hold_current: 0.300
+stealthchop_threshold: 999999
+########################################
+
+########################################
+# Heater Bed Pins
+########################################
+[heater_bed]
+heater_pin: PD7
+sensor_type: Generic 3950
+sensor_pin: PA1
+control: pid
+pid_Kp: 54
+pid_Ki: 0.77
+pid_Kd: 900
+min_temp: 0
+max_temp: 110
+
+########################################
+# Printer Configuration
+########################################
+[printer]
+kinematics: cartesian
+max_velocity: 200
+max_accel: 1000
+max_z_velocity: 5
+max_z_accel: 100
+
+########################################
+# Probe configuration
+########################################
+[probe]
+pin: ^!PE4
+z_offset: 0.0
+x_offset: 0.0
+y_offset: 0.0
+speed: 10.0
+samples: 2
+samples_result: average #median
+sample_retract_dist: 2.0
+samples_tolerance: 0.2
+
+[safe_z_home]
+home_xy_position: 155,155   #140,169
+speed: 100
+z_hop: 5
+z_hop_speed: 5
+
+[output_pin probe_enable]
+pin: PE5
+value: 1
+
+########################################
+# Bed Mesh configuration               # exemple off bed mesh config
```

Comment:
Since bed mesh is a part of the general configuration steps it should be left out of printer example files.

Reply:
The biqu b1 se plus has automatic leveling as standard.  I left it commented because it can be very useful for those who will use it....

Reply:
It will be useful and I expect they will benefit, but bed mesh is already in the startup documentation under the installation and configuration section, so to minimize the amount of information someone needs to get the printer moving correctly leaving out information that will come along later will help minimize the learning curve.  May be best to get another opinion on this one.  I do see the viewpoint of value added that you are expressing.